tools/gulp/tasks/screenshots.ts

```diff
@@ -34,6 +34,9 @@ const FIREBASE_IMAGE = `${TEMP_FOLDER}/screenshot/images`;
 const FIREBASE_DATA_GOLDENS = `screenshot/goldens`;
 const FIREBASE_STORAGE_GOLDENS = 'goldens';
 
+const lastActionTimeout = 1000 * 60 * 6;
+const lastActionRefreshInterval = 1000 * 45;
```

Comment:
Add comments with the human-readable times?

Reply:
Sure. Done